logstash-core-event-java/src/main/java/org/logstash/Event.java

```diff
@@ -76,9 +76,19 @@ public Event(Map data)
         this.metadata_accessors = new Accessors(this.metadata);
 
         this.cancelled = false;
-        this.timestamp = initTimestamp(data.get(TIMESTAMP));
+
+        Object provided_timestamp = data.get(TIMESTAMP);
```

Comment:
so yes, obviously, to clarify, all (my) Java code uses the Java camelCase convention, this is an oversight dur to switching between both languages.